ionic-template/views/SubSectionShow.jsx

```diff
@@ -9,10 +9,9 @@ import { IonItem, IonLabel, IonList, IonListHeader } from '@ionic/react'
 // import * as apiActions from '@/main/apiActions.js'
 
 const propTypes = {}
-const defaultProps = {}
 
 function %SubSection%Show () {
-  // const { id } = useParams()
+  const { id } = useParams()
```

Comment:
This was a bug. Newly generated subsystems would not compile because `id` was undefined.